mlflow/entities/span.py

```diff
@@ -10,10 +10,12 @@
 from opentelemetry.trace import Span as OTelSpan
 
 import mlflow
+from mlflow.entities.assessment import set_pb_value
```

Comment:
Could we import from mlflow.utils.proto_json_utils since that's where this function is defined